github-status/handler.go

```diff
@@ -77,25 +77,41 @@ func Handle(req []byte) string {
 	}
 
 	for _, commitStatus := range status.CommitStatuses {
-		err := reportStatus(commitStatus.Status, commitStatus.Description, commitStatus.Context, &status.EventInfo)
+		err := reportCheck(&commitStatus, &status.EventInfo)
 		if err != nil {
 			log.Fatalf("failed to report status %v, error: %s", status, err.Error())
 		}
 	}
 
-	if val, exists := os.LookupEnv("debug_token"); exists {
-		if val == "true" {
-			log.Printf("Token: %s for Installation: %d", token, status.EventInfo.InstallationID)
-		}
-	}
-
 	// marshal token
 	token = sdk.MarshalToken(token)
 
 	// return auth token so that it can be reused form a same function
 	return token
 }
 
+func getLogs(status *sdk.CommitStatus, event *sdk.Event) (string, error) {
+	client := &http.Client{}
+	var err error
+	gatewayURL := os.Getenv("gateway_url")
+	// TODO: support logs for different commit status contexts
```

Comment:
Please can you explain what you mean?

Reply:
Sorry for my delayed response on this, currently the `stack_deploy` check has no logs. Although for a failure like no/invalid `stack.yml` file or git checkout error the developer needs to debug manually. It will be great to get those logs in `pipeline_logs` as well and put it under `stack_deploy`